templates/citizen_action/_layout.html.twig

```diff
@@ -6,6 +6,30 @@
 
 {% block javascripts %}
     {{ parent() }}
+    <script type="text/javascript">
+        Kernel.onLoad(function(di) {
+            {% if is_granted('ROLE_ADHERENT') %}
+            var unregistrationButton = dom('#citizen_action-unregistration');
+
+            unregistrationButton && on(unregistrationButton, 'click', function () {
```

Comment:
The button should always exist in that case thanks to twig condition. Otherwise it means that something is wrong.

Reply:
Did you forget to change this one ?